cli/cache/node.rs

```diff
@@ -12,26 +12,12 @@ use super::FastInsecureHasher;
 
 pub static NODE_ANALYSIS_CACHE_DB: CacheDBConfiguration =
   CacheDBConfiguration {
-    table_initializer: concat!(
-      "CREATE TABLE IF NOT EXISTS cjsanalysiscache (
+    table_initializer: "CREATE TABLE IF NOT EXISTS cjsanalysiscache (
       specifier TEXT PRIMARY KEY,
```

Comment:
@dsherret are you fine with changes to the cache?

Reply:
Looks good to me. I think the index being removed is fine because it's automatically created.